packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2832,6 +2832,7 @@ export default mixins(
 			async loadNodeTypes (): Promise<void> {
 				const nodeTypes = await this.restApi().getNodeTypes();
 				this.$store.commit('setNodeTypes', nodeTypes);
+				this.$store.commit('setCommunityNodeFlags');
```

Comment:
why do we need a separate mutation for this? should be simply done when it's added